std/algorithm/searching.d

```diff
@@ -2201,6 +2201,21 @@ template canFind(alias pred="a == b")
     assert(canFind([0, 1, 2, 3], [1, 3], [2, 4]) == 0);
 }
 
+/**
+ * Example using a custom predicate.
+ * Note that the needle appears as the second argument of the predicate.
+ */
+@safe unittest
+{
+    auto words1 = [
```

Comment:
I suppose that it's a bit nitpicky, by why `words1` and not `words`? The `1` implies that there's another `words`, which there isn't.